recipes/busco/build.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+mkdir -p $PREFIX/bin/
+cp BUSCO_v1.2.py $PREFIX/bin
+ln -s $PREFIX/bin/BUSCO_v1.2.py $PREFIX/bin/BUSCO_v1.1b1.py
```

Comment:
Sure! Apparently the `BUSCO` package comes with an executable that includes the version in the name. The `dammit` package in turn depends on a specific `BUSCO` executable. So this is how I solved that. Another way, which might be more elegant, is to symlink the exutable to a version independent name (ie `BUSCO.py`) and than patch the `dammit` code to run this executable. I admit to taking the easy way out here...